be/src/exec/hash_join_node.cpp

```diff
@@ -195,6 +195,19 @@ void HashJoinNode::_init_hash_table_param(HashTableParam* param) {
     param->output_tuple_column_timer = _output_tuple_column_timer;
     param->output_slots = _output_slots;
 
+    std::set<SlotId> predicate_slots;
+    for (ExprContext* expr_context : _conjunct_ctxs) {
+        std::vector<SlotId> expr_slots;
+        expr_context->root()->get_slot_ids(&expr_slots);
+        predicate_slots.insert(expr_slots.begin(), expr_slots.end());
+    }
+    for (ExprContext* expr_context : _other_join_conjunct_ctxs) {
+        std::vector<SlotId> expr_slots;
+        expr_context->root()->get_slot_ids(&expr_slots);
+        predicate_slots.insert(expr_slots.begin(), expr_slots.end());
+    }
+    param->predicate_slots = predicate_slots;
```

Comment:
```suggestion
    param->predicate_slots = std::move(predicate_slots);
```
Use move assignment for better performance

Reply:
Ok.